Reflection/ReflectionProperty.php

```diff
@@ -29,13 +29,6 @@ class ReflectionProperty implements Reflector
     #[LanguageLevelTypeAware(['8.1' => 'string'], default: '')]
     public $class;
 
-    /**
-     * @var bool
```

Comment:
Why have you removed `readonly` of property?

Reply:
Oops, sorry for the noise. You are completely right. Probably we need a test to check such cases at the end.

Reply:
Added this to the list